src/us/kbase/test/auth2/providers/GlobusIdentityProviderTest.java

```diff
@@ -284,7 +320,7 @@ public void returnsBadAudience() throws Exception {
 		final String bauth = getBasicAuth(testIDConfig);
 		final String authCode = "foo2";
 		final String authtoken = "footoken";
-		setUpCallAuthToken(authCode, authtoken, redir, bauth);
+		setUpCallAuthToken(authCode, "pixydust", authtoken, redir, bauth);
```

Comment:
pixy?

Reply:
https://www.oauth.com/oauth2-servers/pkce/
> Proof Key for Code Exchange (abbreviated PKCE, pronounced “pixie”)

Reply:
Just a misspelling that I'm not going to fix